src/dodal/devices/i22/dcm.py

```diff
@@ -74,63 +61,45 @@ def __init__(
 
         # Soft metadata
         # If supplied include crystal details in output of read_configuration
-        crystal_1_metadata = crystal_1_metadata or CrystalMetadata()
-        crystal_2_metadata = crystal_2_metadata or CrystalMetadata()
         with self.add_children_as_readables(ConfigSignal):
             if crystal_1_metadata.usage is not None:
```

Comment:
Should: If we've now removed the option for these to be `None` then I don't think we need to check them all?

Reply:
removed